app/src/main/res/layout/media_player_view.xml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this view still in use? If no it should be removed and v2 renamed to this

Reply:
@shubertm Removed it.